packages/react-jss/src/withStyles.js

```diff
@@ -39,6 +39,29 @@ const getStyles = <Theme: {}>(styles: Styles<Theme>, theme: Theme) => {
   if (typeof styles !== 'function') {
     return styles
   }
+  if (process.env.NODE_ENV !== 'production') {
+    if (styles.length < 1) {
+      console.warn(
+        `
+        Warning: [JSS]: This Component uses themed styles notation (function) while not relying on a theme (0 arguments).
+        It slows your app down and you should rewrite these styles to plain object notation.
+      `
+          .replace(/[\s]{2,}/gim, ' ')
```

Comment:
Why do we need that regexp?

Reply:
it is multiline message so it will keep all indentation when being printed without adjusting

Reply:
I think this is something that should be evtl done in warning module

Reply:
or keep errors short